pandas/io/parsers.py

```diff
@@ -101,7 +100,8 @@ class ParserWarning(Warning):
 keep_default_na : bool, default True
     If na_values are specified and keep_default_na is False the default NaN
     values are overridden, otherwise they're appended to
-parse_dates : boolean, list of ints or names, list of lists, or dict, default False
+parse_dates : boolean, list of ints or names, list of lists, or dict,
+    default False
```

Comment:
Yes, if the options are clearly listed below (as is already a bit the case, only the bullet symbols are missing), a shorter type description is certainly OK. Eg `boolean, list or dict, default False` is certainly sufficient (and short enough)